crds/core/selectors.py

```diff
@@ -2035,16 +2035,17 @@ def _validate_raw_key(self, key, valid_values_map):
         for other in self.keys():
             if key != other and match_superset(other, key) and \
                 not different_match_weight(key, other):
-                warn = log.verbose_warning if self._merge_overlaps else log.warning
-                warn("-"*40 + "\nMatch case\n",
-                     log.PP(self.match_item(key)),
-                     "\nis an equal weight special case of\n",
-                     log.PP(self.match_item(other)), """
-For some parameter sets, CRDS interprets both matches as equally good.
-See the file submission section of the CRDS server user's guide here:
-    https://jwst-crds.stsci.edu/static/users_guide/index.html
-for more explanation.
-""", "-"*40)
+                # if self._merge_overlaps:
```

Comment:
Unless there is some critical future need, which should be explained, there is no need to keep commented code.

Reply:
the future need was me remembering to ask about the additional condition `if self._merge_overlaps`... will remove comments and resubmit